src/components/Post/GovernanceSideBar/index.tsx

```diff
@@ -694,6 +936,59 @@ const GovernanceSideBar: FC<IGovernanceSidebarProps> = (props) => {
+																		<span className='h-[25px]'>{lastVote?.decision === EVoteDecisionType.AYE ? <p><AyeGreen /> <span className='capitalize font-medium text-[#2ED47A]'>{'Aye'}</span></p> :lastVote?.decision === EVoteDecisionType.NAY ?  <div><DislikeIcon className='text-[#F53C3C]'/> <span className='mb-[5px] capitalize font-medium text-[#F53C3C]'>{'Nay'}</span></div> : lastVote?.decision === EVoteDecisionType.SPLIT  ? <p><SplitYellow className='mb-[-2px]'/> <span className='capitalize font-medium text-[#FFBF60]'>{'Split'}</span></p>  : lastVote?.decision === EVoteDecisionType.ABSTAIN  ? <p className='flex justify-center align-middle'><AbstainGray className='mr-1 mb-[-8px]'/> <span className='capitalize font-medium  text-[#243A57]'>{'Abstain'}</span></p>: null }</span>
+																	</Tooltip>
+																	<Tooltip placement="bottom"  title="Time"  color={'#E5007A'} className=''>
+																		<span className=''><ClockCircleOutlined className='mr-1' />{dayjs().format('Do MMM \'YY')}</span>
+																	</Tooltip>
+
+																	<Tooltip placement="bottom"  title="Amount"  color={'#E5007A'}className=''>
+																		<span>
+																			<MoneyIcon className='mr-1'/>
+																			{formatedBalance(lastVote?.balance.toString(), unit)}{` ${unit}`}
+																		</span>
+																	</Tooltip>
+
+																	<Tooltip placement="bottom"  title="Conviction"  color={'#E5007A'} className='ml-[-5px]'>
+																		<span title='Conviction'>
+																			<ConvictionIcon className='mr-1'/>
+																			{lastVote?.conviction}x
+																		</span>
+																	</Tooltip>
+																</div>
+															</div> : voteCount > 0 ?
+																<Spin spinning={ isLastVoteLoading } indicator={<LoadingOutlined />}>
+																	<div className=''>
+																		<p className='font-medium text-[12px] leading-6 text-[#243A57] mb-[5px]'>Last Vote:</p>
+																		<div className='flex justify-between text-[#243A57] text-[12px] font-normal leading-6 mb-[-5px]'>
+																			<Tooltip placement="bottom"  title="Decision"  color={'#E5007A'} className='max-w-[100px] max-[345px]:w-auto'>
+																				<span className='h-[25px]'>{vote.decision == 'yes' ? <p><AyeGreen /> <span className='capitalize font-medium text-[#2ED47A]'>{'Aye'}</span></p> :vote.decision == 'no' ?  <div><DislikeIcon className='text-[#F53C3C]'/> <span className='mb-[5px] capitalize font-medium text-[#F53C3C]'>{'Nay'}</span></div> : vote.decision == 'abstain' && (!(vote.balance as any).abstain)  ? <p><SplitYellow className='mb-[-2px]'/> <span className='capitalize font-medium text-[#FFBF60]'>{'Split'}</span></p>  : vote.decision == 'abstain' && (vote.balance as any).abstain ? <p className='flex justify-center align-middle'><AbstainGray className='mr-1 mb-[-8px]'/> <span className='capitalize font-medium  text-[#243A57]'>{'Abstain'}</span></p>: null }</span>
```

Comment:
Please extract this span to a seperate component

Reply:
will fix